connect/client.go

```diff
@@ -24,3 +63,25 @@ func IsRegistered() bool {
 func URLDefault() bool {
 	return CFG.BaseURL == defaultBaseURL
 }
+
+func printInformation(action string) {
```

Comment:
This would also be a good candidate for a separate PR. I could use it directly in my Deregister code.

Reply:
I will split it out

Reply:
done